src/model/channel/guild_channel.rs

```diff
@@ -120,6 +129,7 @@ pub struct GuildChannel {
     /// An approximate count of messages in the thread, stops counting at 50.
```

Comment:
Could update this "An approximate count of messages in the thread. Currently saturated at 255 (this limitation is removed in the next breaking serenity version)."